platforms/web/lib/useWysiwyg.test.tsx

```diff
@@ -31,10 +31,57 @@ const Editor = forwardRef<HTMLDivElement>(function Editor(props, forwardRef) {
     contentEditable={isWysiwygReady} />;
 });
 
+function toContainHtml(
+    // TODO: remove lint ignores? I can't find the right import to be able to use the types.
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    this: any, /* MatcherState */
```

Comment:
I'm afraid, that we need to add the jest types and use `jest.MatcherContext` here

Reply:
Thanks, how would I do that?

Reply:
// Adding jest type dev dependencies
- `yarn add -D @types/jest`
// Add jest in the global types
- Add `jest`in the `types`array in the tsconfig
// Use the MatcherContext type for the `this` type
- `this: jest.MatcherContext`

Reply:
Thank you!

Reply:
Done!